homeassistant/components/eps/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""EPS integration."""
+import asyncio
+
+from pyepsalarm import EPS
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_TOKEN, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import EPSDataUpdateCoordinator
+
+PLATFORMS = [Platform.ALARM_CONTROL_PANEL]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up EPS config."""
+    token: str = entry.data[CONF_TOKEN]
+    username: str = entry.data[CONF_USERNAME]
+    password: str = entry.data[CONF_PASSWORD]
```

Comment:
What happens if the password is changed? I think you should add a reauth flow to handle that already in this PR.